src/libraries/Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/src/CultureReplacer.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Threading;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    public class CultureReplacer : IDisposable
```

Comment:
We already have such a helper: https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/ThreadCultureChange.cs